### DISCUSSION TITLE
Sharing my two Bat files for an easy use. (Send to)

### DISCUSSION THREAD

**Discussion by WaDn-X** [xinntao/Real-ESRGAN#571] [Show and tell]
Hello everyone,

I Make two Bat files for the Portable realesrgan-ncnn-vulkan.exe Version
Just place them inside the dir where "realesrgan-ncnn-vulkan.exe" is inside.
now your can drag your Picture on one of them and be done.

I also make links to them in my Windows Send to ( Win + R, enter "shell:SendTo" for easy use.

I hope that's helpful for you too.

Have a Nice Day
[Easy Use - For Real-ESRGAN.zip](https://github.com/xinntao/Real-ESRGAN/files/10743592/Easy.Use.-.For.Real-ESRGAN.zip)

**Comment by Pandafluff025** [comment]
UR THE GOAT !!!!!
